logger.js

```diff
@@ -147,9 +153,20 @@ function pinoLogger (opts, stream) {
       }
 
       if (shouldLogSuccess) {
-        if (receivedMessage !== undefined) {
+        const shouldLogReceived = receivedMessage !== undefined || onRequestReceivedObject !== undefined
+
+        if (shouldLogReceived) {
           const level = getLogLevelFromCustomLogLevel(customLogLevel, useLevel, res, undefined, req)
-          req.log[level](receivedMessage(req, res))
+          const receivedObjectResult = onRequestReceivedObject !== undefined ? onRequestReceivedObject(req, res, undefined) : undefined
+          const receivedStringResult = receivedMessage !== undefined ? receivedMessage(req, res) : undefined
+
+          if (receivedObjectResult && receivedStringResult) {
+            req.log[level](receivedObjectResult, receivedStringResult)
+          } else if (receivedStringResult) {
+            req.log[level](receivedStringResult)
+          } else {
+            req.log[level](receivedObjectResult)
+          }
```

Comment:
There are quite a a lot of conditionals in this block. Could you simplify them a bit and remove some duplications?